docs/unity/part-1.md

```diff
@@ -105,6 +112,8 @@ At this point you should have the single player game working. In your CLI, your
 spacetime start
 ```
 
+💡Below examples Rust language, [but you may also use C#](../modules/c-sharp/index.md) at the cost of 1/2 the speed. 
```

Comment:
I'm not sure that we should advertise/codify this speed difference :thinking: it's kind of in flux. @jdetter do you have thoughts?

Reply:
There's a chance it could sound like we're passively-aggressively pushing them towards Rust, but at the same time... this would probably explain why a Unity tutorial shows Rust server code instead of C#?

Reply:
For now, I removed the speed memo, but left the C# link